### PR TITLE
New option for summary program

### PR DESCRIPTION
**Pull request by tskille**
Enable output of summary vectors without any header info.

### CODE REVIEW

File: test_util/summary.cpp [hunk 1]
```diff
@@ -39,6 +39,7 @@ static void printHelp() {
               << "\nIn addition, the program takes these options (which must be given before the arguments):\n\n"
               << "-h Print help and exit.\n"
               << "-l list all summary vectors.\n"
+              << "-n print summary vectors without no headers.\n"
```
Comment (akva2) [inline test_util/summary.cpp:42]: without or with no.

File: test_util/summary.cpp [hunk 2]
```diff
@@ -77,15 +78,19 @@ int main(int argc, char **argv) {
     int c                          = 0;
     bool reportStepsOnly           = false;
     bool listKeys                  = false;
+    bool noHeaders                 = false;
```
Comment (akva2) [inline test_util/summary.cpp:81]: this is mainly a preference thing but imho it is much more intuitive with enable than disable bools. ie bool headers = true, -n makes it false, no negation in ifs below.

File: test_util/summary.cpp (hunk 1 above)
Comment (tskille) [reply test_util/summary.cpp]: fixed, should be without.

File: test_util/summary.cpp (hunk 2 above)
Comment (tskille) [reply test_util/summary.cpp]: I fully agree with you. Copy past without thinking is not allways a good thing. Fixed now.

**Comment by akva2** [comment]
jenkins build this no downstreams please

**Review by akva2 — APPROVED** [review]

**Pull request merged**